### PR TITLE
Sign and log forms

### PR DESCRIPTION
**Pull request by royemosby**
Shifted Michael's code to the correct directories

- added react-router-dom
- created Nav, Home, SignupForm, LoginForm components
- removed bootstrap

### CODE REVIEW

File: src/Components/SignupForm.jsx
Comment (Miminiverse) [inline src/Components/SignupForm.jsx:1]: I saw on the backend that they requested names, emails, and passwords to sign up, but we only require names and passwords for the frontend, is there any reason for that?

Comment (royemosby) [reply src/Components/SignupForm.jsx:1]: @mikelikecoding ^^

Comment (mikelikecoding) [reply src/Components/SignupForm.jsx:1]: I will check and correct it

**Comment by anniethewebcoder** [comment]
Is it connected to our backend? If not, then it works on my end.

**Review by anniethewebcoder — APPROVED** [review]
I looked over the code and ran it. I noticed that it's not connected to our backend but the frontend feedback works good.

**Pull request merged**